### DISCUSSION TITLE
Upgrade the version?

### DISCUSSION THREAD

**Discussion by shlroland** [micromark/micromark#177] [Q&A] [answered]
Several PRs related to this issue <https://github.com/micromark/micromark-extension-gfm/issues/12> have been merged and a new version has been released. Should we also consider upgrading the version of `micromark-extension-gfm`? Additionally, there are related packages such as `mdast-util-gfm` and `remark-gfm`.

@wooorm

**Comment by wooorm** [comment, marked as answer]
Hey!

> Should we also consider upgrading the version of `micromark-extension-gfm`?

Not needed! You can run `npm update` locally and you get the latest versions.
Even if we did that, you’d still have to run `npm update` locally.

We prefer being really lax about version numbers. We follows semver. So, except when things definitely don‘t work, we use `^x.0.0`
